<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>757</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QSL Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTableWidget" name="tableWidget">
        <property name="editTriggers">
         <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
        </property>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="currentCallLabel">
          <property name="font">
           <font>
            <pointsize>22</pointsize>
           </font>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Sort and Filter</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>QSO Date From</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QToolButton" name="toolButton">
             <property name="text">
              <string>X</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1" colspan="3">
            <widget class="QComboBox" name="bandComboBox">
             <property name="editable">
              <bool>true</bool>
             </property>
             <item>
              <property name="text">
               <string>All Bands</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>160m</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>80m</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>60m</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>40m</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>30m</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>20m</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>18m</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>15m</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>12m</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>10m</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>6m</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2m</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>70cm</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>23cm</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>13cm</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Mode</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1" colspan="3">
            <widget class="QComboBox" name="modeComboBox">
             <property name="editable">
              <bool>true</bool>
             </property>
             <item>
              <property name="text">
               <string>All Modes</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>AM</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>CW</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>FM</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>FT4</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>FT8</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>SSB</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>SSTV</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="date1LineEdit">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Band</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>to</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QPushButton" name="addQSOPushButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Add QSO...</string>
             </property>
             <property name="default">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="5" column="2" colspan="2">
            <widget class="QPushButton" name="pushButton_2">
             <property name="text">
              <string>Set Sort and Filter</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Call Sign</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="callLineEdit">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QLineEdit" name="date2LineEdit">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Sort Order</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1" colspan="2">
            <widget class="QComboBox" name="sortOrderComboBox">
             <item>
              <property name="text">
               <string>QSO ID</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>QSO Date Time</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Call Sign</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="4" column="3">
            <widget class="QCheckBox" name="descendCheckBox">
             <property name="text">
              <string>Desending</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>Import from ADIF File...</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="pushButton_4">
            <property name="text">
             <string>Export to ADIF File...</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QPushButton" name="pushButton_3">
            <property name="text">
             <string>Log out Current Call Sign</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="pushButton_5">
            <property name="text">
             <string>Statistics...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>757</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>tableWidget</tabstop>
  <tabstop>callLineEdit</tabstop>
  <tabstop>date1LineEdit</tabstop>
  <tabstop>date2LineEdit</tabstop>
  <tabstop>bandComboBox</tabstop>
  <tabstop>modeComboBox</tabstop>
  <tabstop>sortOrderComboBox</tabstop>
  <tabstop>descendCheckBox</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>addQSOPushButton</tabstop>
  <tabstop>toolButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
